cmd/pilot-agent/main.go

```diff
@@ -111,9 +115,13 @@ func init() {
 		"Sidecar proxy unique ID. If not provided uses ${POD_NAME}.${POD_NAMESPACE} environment variables")
 	proxyCmd.PersistentFlags().StringVar(&sidecar.Domain, "domain", "cluster.local",
 		"DNS domain suffix")
+	proxyCmd.PersistentFlags().StringVar(&customConfig, "envoyconfig", "",
+		"Custom envoy config")
 
 	cmd.AddFlags(rootCmd)
 
+	rootCmd.Flags().AddGoFlagSet(flag.CommandLine)
```

Comment:
Not sure what you mean - I want to have all the standard go flags available, no reason to not support them - AddGoFlagSet is on Flags()